<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1757</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>NISN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>721</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rosewood Std Regular</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pendaftaran PMB Universitas Muhammadiyah Cirebon</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNISN">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>70</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNama">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>120</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nama Lengkap</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLahir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tanggal Lahir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSekolah">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>94</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Asal Sekolah</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOrgtua">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>280</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAlamat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTelepon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Telepon</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>53</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alamat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nama Ibu/Ayah</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSimpan">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>569</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mahameru-Bold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnSimpan{
	color: rgb(150, 155, 125);
	background-color: rgb(0, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCari">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cari</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHapus">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>569</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mahameru-Bold</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnHapus{
	color: rgb(125, 155, 255);
	background-color: rgb(255, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Dalete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>570</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mahameru-Bold</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnClear{
	color: rgb(12, 155, 255);
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="gridPendaftaran">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>841</width>
      <height>321</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>11</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QRadioButton" name="optPerempuan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Perempuan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optLaki">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Laki-laki</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jenis Kelamin</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboProdi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TEKNIK INFORMATIKA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ILMU HUKUM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KEDOKTERAN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PENDIDIKAN AGAMA ISLAM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SASTRA INGGRIS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TEKNIK INDUSTRI</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Program Studi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>470</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>@2022 Kelompok 6 - TIF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1757</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
